install_scripts/templates/common/kubernetes.sh

```diff
@@ -255,6 +255,8 @@ airgapLoadKubernetesCommonImages1111() {
     docker tag b2b03e9146e1 docker.io/registry:2
     docker tag 6521ac58ca80 docker.io/envoyproxy/envoy-alpine:v1.6.0
     docker tag 6a9ec4bcb60e gcr.io/heptio-images/contour:v0.5.0
+
+    docker load < rook.tar
```

Comment:
where does this come from?

Reply:
Chatops includes it, but I can't remember any good reason it shouldn't be in the common package. I'm going to move it there.